net-p2p/syncthing/syncthing-0.14.12.ebuild

```diff
@@ -17,6 +17,8 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86 ~arm"
 IUSE="tools"
 
+DEPEND="tools? ( >=dev-lang/go-1.7.1:= )"
```

Comment:
The point of that is to improve descriptions, not to hide shitty descriptions behind other use flags so that I won't notice!